app/components/footer/talk_to_us_component.html.erb

```diff
@@ -17,7 +17,7 @@
                 <p class="telephone">0800 389 2500</p>
               </div>
             </div>
-            <p>Monday to Friday between <strong>8:30am and 5:30pm</strong></p>
+            <p>Call or chat to us Monday to Friday, <strong>8:30am until 5:30pm</strong>, except on <a href="https://www.gov.uk/bank-holidays">bank holidays</a>.</p>
```

Comment:
```suggestion
            <p>Call or chat to us online Monday to Friday, <strong>8:30am until 5:30pm</strong>, except on <a href="https://www.gov.uk/bank-holidays">bank holidays</a>.</p>
```
Do we just need to be a bit more explicit here that we're referring to the live chat service, too? (just 'chat to us' could still imply that we're just referring to the phone line)